department-of-veterans-affairs/vets-website
# Update faster claim processing page

## Summary
Update 'Faster claim processing' page on the Pension Benefits form (21P-527EZ)

## How to run in local environment
1. Check out this branch locally
2. In your local environment, set the `pension_form_enabled` flipper to 'enabled' at http://localhost:3000/flipper/features/pension_form_enabled
3. Run `vets-website` and `vets-api`

## How to verify
1. Go to `http://localhost:3001/pension/application/527EZ` in your browser
2. Go to the 'Faster claim processing' page in the 'Additional information' chapter
3. Verify the yesNo widget has been replaced with the yesNo web component pattern

## Acceptance criteria
[Issue in Github - 76772](https://github.com/department-of-veterans-affairs/va.gov-team/issues/76772)

## Related issue(s)
n/a

## Testing done

## Screenshots
| Before | After |
| ------ | ------ |
|![localhost_3001_pension_application_527EZ_form-saved (12)](https://github.com/department-of-veterans-affairs/vets-website/assets/7394764/68cb42db-b0df-4086-bb29-bae3846a6d00)|![localhost_3001_pension_application_527EZ_form-saved (13)](https://github.com/department-of-veterans-affairs/vets-website/assets/7394764/742116be-5c4c-4bbe-9b9e-69a8562e1665)|

## What areas of the site does it impact?
Pension Benefits Application

### Quality Assurance & Testing

### Error Handling


### Authentication
